src/state/slices/tradeInputSlice/tradeInputSlice.ts

```diff
@@ -78,10 +78,10 @@ export const tradeInput = createSlice({
 
       state.sellAsset = action.payload
     },
-    setSellAssetAccountNumber: (state, action: PayloadAction<AccountId | undefined>) => {
+    setSellAssetAccountId: (state, action: PayloadAction<AccountId | undefined>) => {
```

Comment:
Naming was off here, this doesn't set an AccountNumber but an AccountId, same for the below

Reply:
Great catch!